docs/client_advanced.rst

```diff
@@ -56,6 +56,14 @@ For *text/plain* ::
 
     await session.post(url, data='Привет, Мир!')
 
+.. note::
+   ``Authorization`` header will be removed if you get redirected
+   to a different host or protocol.
```

Comment:
I think this should mention the redirect case, not just in the versionchanged directive.

Reply:
Ok, moved to the note.

Reply:
I'd have a copy, I think. Not a move. Or not a copy but rather some mention about when things changed.